src/ExceptionReporting/ExceptionReportInfo.cs

```diff
@@ -193,6 +193,13 @@ public bool ShowAssembliesTab
 		/// </summary>
 		public string EmailReportAddress { get; set; } = "";
 
+		/// <summary>
+		/// Email object
```

Comment:
This is a minor left over - should be "Email Subject" - but I won't let this stop the pull request. This comment will hopefully remind me about it